Revision: d51b46d2b6aa5e96d53c614f4bfba0e6a1c9e3d6
Patch-set: 1
File: runtime/entrypoints/entrypoint_utils-inl.h

152:6-152:97
Tue Jul 26 01:02:23 2016 +0000
Author: Andreas Gampe <1041833@85c56323-6fa9-3386-8a01-6480fb634889>
UUID: 6a4bd746_59b754d3
Bytes: 52
Consider ThrowIllegalAccessError from common_throws.

File: runtime/verifier/method_verifier.cc

1263:9-1263:39
Tue Jul 26 01:02:23 2016 +0000
Author: Andreas Gampe <1041833@85c56323-6fa9-3386-8a01-6480fb634889>
UUID: 6a4bd746_39e8d0aa
Bytes: 149
That's the first time we use it outside of lambdas. This error wasn't meant to be used outside of testing. Not sure we have something better, though.

